spec/helpers/views_helper_spec.rb

```diff
@@ -0,0 +1,31 @@
+require "rails_helper"
+
+RSpec.configure do |config|
+  config.include AbstractController::Translation
+end
+
+describe ViewsHelper, type: :helper do
+  include CasesHelper
+
+  describe "#get_headings" do
+    it "text for rejected case type heading" do
```

Comment:
The test name should read like a sentence. eg. it "returns text for rejected case type heading"